### PR TITLE
.spec add (network_parse,server,types), synproxy plugin

### PR DESCRIPTION
**Pull request by DefauIt**
added Installed (but unpackaged) file(s) in .spec (RPM)
synproxy disabled by default

### CODE REVIEW

File: contrib/redhat/collectd.spec [hunk 1]
```diff
@@ -16,7 +16,7 @@
 #
 # - fetch the desired collectd release file from https://collectd.org/files/
 #   and save it in your ~/rpmbuild/SOURCES/ directory (or build your own out of
-#   the git repository: ./build.sh && ./configure && make dist-bzip2)
+#   the git repository: ./build.sh && ./configure && make-dist-bz2)
```
Comment (octo) [inline contrib/redhat/collectd.spec]: Please rebase on *master* since this reverts #2459.

Comment (XakRu) [reply contrib/redhat/collectd.spec]: error in jenkins comes from https://github.com/collectd/collectd/commit/ea341582f90a9a7ee4b5525551edc2fdb4ec1226
Build #988 (Sep 27, 2017 8:25:07 AM)

**Comment by rubenk** [comment]
Please bump the Release number in the spec accordingly and make sure it matches in the %changelog section.

**Comment by mfournier** [comment]
Merged as a4f7a5d. Thanks !

**Pull request closed without merging**